src/commands/init.ts

```diff
@@ -1,73 +1,109 @@
 import * as vscode from 'vscode'
+import fs from 'fs'
 import { loader } from '../modules/loader'
 import { npmInstall } from '../modules/npm'
-import { getTemplates, getTypeOptions, ProjectType } from '../modules/project'
+import {
+  getSdkOptions,
+  getTemplates,
+  getProjectTypeOptions,
+  ProjectType,
+  SdkVersion,
+} from '../modules/project'
 import { bin } from '../modules/bin'
 import { track } from '../modules/analytics'
+import { pick } from '../modules/pick'
 
-export async function init(type?: ProjectType) {
-  const options = getTypeOptions()
-
-  let option: {
-    type: ProjectType
-    name: string
-  } | null = type
-    ? options.find((option) => option.type === type) || null
-    : null
-  let template = ''
-
-  if (!option) {
-    const selected = await vscode.window.showQuickPick(
-      options.map((option) => option.name),
-      {
-        ignoreFocusOut: true,
-        title: 'Create Project',
-        placeHolder: 'Select the project type',
+export async function init() {
+  const sdkOptions = getSdkOptions()
+  const selectedSdk = await pick(sdkOptions, 'name', {
+    ignoreFocusOut: true,
+    title: 'Create Project',
+    placeHolder: 'Select the SDK version',
+  })
+
+  if (!selectedSdk) {
+    return
+  }
+
+  track(`decentraland.commands.init:select_sdk_version`, {
+    version: selectedSdk.version,
+  })
+
+  switch (selectedSdk.version) {
+    case SdkVersion.SDK6: {
+      const params = await getSdk6InitParams()
+      if (!params) {
+        return
       }
-    )
-    if (!selected) {
-      return
+      await initSdk6(params.type, params.templateUrl)
+      break
+    }
```

Comment:
maybe we should default to `SdkVersion.SDK6` until we release the sdk7?

```ts
case SdkVersion.SDK7: {
  await initSdk7()
  break
}
case SdkVersion.SDK6: 
case default: {
  const params = await getSdk6InitParams()
  if (!params) {
    return
  }
  await initSdk6(params.type, params.templateUrl)
  break
}
```

Reply:
I didn't add a default case because `selectedSdk` can't be undefined (if the user selects nothing, we exit the init flow, in [line 24](https://github.com/decentraland/editor/pull/90/files#diff-ec8485d3117f43c2d2d8e3ec7b2830f2d6b5df3f20626a62b04f61ccd70ec93fR24)).